js/metronome.js

```diff
@@ -1,67 +1,65 @@
 // ------------------------------------------------------------
 // Metronome functionality
 // ------------------------------------------------------------
-var $downbeat_beep = new Audio("audio/metronome/downbeat_beep.mp3");
-var $beat_beep = new Audio("audio/metronome/beat_beep.mp3");
+const $downbeat_beep = new Audio("audio/metronome/downbeat_beep.mp3");
+const $beat_beep = new Audio("audio/metronome/beat_beep.mp3");
 
-var $beats_per_measure = 4;
-var $current_beat = 0;
+let $beats_per_measure = 4;
+let $current_beat = 0;
 
-var $bars_per_chord = 0;
-var $current_bar = 1;
+let $bars_per_chord = 0;
```

Comment:
(Same as above.) These shouldn't be declared with let as they are outside the block context and in the global context. These should remain var declared.